Formula/meson.rb

```diff
@@ -0,0 +1,22 @@
+class Meson < Formula
+  desc "Fast and user friendly build system"
+  homepage "http://mesonbuild.com/"
+  url "https://github.com/mesonbuild/meson/releases/download/0.31.0/meson-0.31.0.tar.gz"
+  sha256 "803d7aa99329c5439ef971c1dcaa39de12d9208f82f0ed029e3e8f9c13ef93f0"
+
+  depends_on "python3"
+  depends_on "ninja"
+
+  def install
+    version = Language::Python.major_minor_version("python3")
+    ENV["PYTHONPATH"] = lib+"python#{version}/site-packages"
+
+    system "python3", *Language::Python.setup_install_args(prefix)
+
+    bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/meson.py", "--version"
```

Comment:
Would be nice if this was `meson` rather than `meson.py`

Also, can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!